### PR TITLE
Move to slash commands & standardize author embeds

### PR DESCRIPTION
**Pull request by Phloot**
- prefixed -> slash commands
- standardize embed 
- remove unnecessary dependencies

### CODE REVIEW

File: src/omen/cogs/wvw_commands.py [hunk 1]
```diff
@@ -226,13 +228,15 @@ async def matchup(self, ctx, member: discord.Member = None):
...
             embed.add_field(name="Objectives", value=embed_obj_value_string, inline=False)
-            await ctx.channel.send(files=[author_img_attached, thumb_img_attached], embed=embed)
+            await interaction.response.send_message(files=[author_img_attached, thumb_img_attached], embed=embed)
         except Exception as e:
             print(e)
 
-    @commands.command()
-    async def worldpop(self, ctx, region: to_lower="na", member: discord.Member = None):
+    @app_commands.command(name="worldpop", description="Gets the world population")
+    async def worldpop(self, interaction: discord.Interaction, region: str):
```
Comment (HelloIAmWorld) [inline src/omen/cogs/wvw_commands.py:236]: add validation for region. 

maybe something like 
if region.lower() != "eu" || region.lower() != "na" 
     return await interaction.response.send_message("error")

File: src/omen/configs/config.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "internal": {
-        "version": "v1.1.0"
+        "version": "v1.1.1"
     },
     "authentication": {
         "gw2_token": "84CF4DFD-D23B-9D4D-A94D-918A33DD0F8FA692141E-717C-440B-94EA-485BE23F3D1D"
```
Comment (HelloIAmWorld) [inline src/omen/configs/config.json:6]: public api token?

File: src/omen/cogs/wvw_commands.py (hunk 1 above)
Comment (Phloot) [reply src/omen/cogs/wvw_commands.py]: added options for command + validation

File: src/omen/configs/config.json (hunk 2 above)
Comment (Phloot) [reply src/omen/configs/config.json:6]: read only api token and contains no sensitive info

**Review by HelloIAmWorld — COMMENTED** [review]
lgtm

**Pull request merged**